Carthage Sample/Carthage-sample-app.xcodeproj/project.pbxproj

```diff
@@ -102,7 +102,8 @@
 				05ADED97260E33DB0002E5E3 /* MaskingUseCasesViewController.swift */,
 				05ADED9A260E45BE0002E5E3 /* MaskingViewController.swift */,
 			);
-			path = Masking;
+			name = Masking;
```

Comment:
Can we also move `AppDelegete.swift`, `SceneDelegate.swift` to the shared folder?
Maybe also `Info.plist` `Assets.xcassets` if possible.

Reply:
Yes, those can be Shared as well